### PR TITLE
Update non-document links in EDA code base

### PR DESCRIPTION
**Pull request by xinnanlisf**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

**Review by aarsvoboda — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-**Note: FOR INTERNAL USE ONLY. For support questions, please use the EDA Chatter group in [The Power of Us Hub](https://powerofus.force.com).
-For feature requests, please use the Ideas tab in [The Power of Us Hub](https://powerofus.force.com).**
+**Note: FOR INTERNAL USE ONLY. For support questions, please use the Education Data Architecture group in [the Trailblazer Community](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi4SAE).
```
Comment (salvadorsfdc) [inline .github/ISSUE_TEMPLATE.md:1]: Just a little thing, I noticed the quip doc has this link which has extra parameters:
https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi4SAE?tab=discussion&sort=LAST_MODIFIED_DATE_DESC

The link seems to redirect to the correct page

**Review by salvadorsfdc — CHANGES_REQUESTED** [review]
Looks good, just 1 question

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (xinnanlisf) [reply .github/ISSUE_TEMPLATE.md:1]: let's use this link. I will update the quip doc.

Comment (salvadorsfdc) [reply .github/ISSUE_TEMPLATE.md:1]: thanks!

**Review by salvadorsfdc — APPROVED** [review]
looks good!

**Pull request merged**